### DISCUSSION TITLE
The product interface org.apache.jackrabbit.api.JackrabbitSession annotated with @ProviderType should not be implemented by custom code. Detected in com.adobe.acs.commons.wrap.jackrabbit.JackrabbitSessionIWrap contained in /apps/acs-commons/install/acs-aem-commons-bundle-6.0.6.jar.

### DISCUSSION THREAD

**Discussion by tech-arch-forum** [Netcentric/accesscontroltool#674] [Q&A]
The product interface org.apache.jackrabbit.api.JackrabbitSession annotated with @ProviderType should not be implemented by custom code. Detected in com.adobe.acs.commons.wrap.jackrabbit.JackrabbitSessionIWrap contained in /apps/acs-commons/install/acs-aem-commons-bundle-6.0.6.jar. Any Fixes available in next release?

**Comment by kwin** [comment]
Cross-posting won’t speed up things. Everything related to this is discussed in #671.
